execution_engine/CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ All notable changes to this project will be documented in this file.  The format
 
 ### Changed
 * Update pinned version of Rust to `nightly-2021-06-17`
-
+* Added a `max_associated_keys` configuration option for a hard limit of associated keys under accounts.
```

Comment:
This bullet point should be under a new `### Added` section, and that section should come before the `### Changed` one (see the comment lines near the top of the file for the order and meaning of sections).